generators/entity-client/templates/react/src/test/javascript/spec/app/entities/entity-reducer.spec.ts.ejs

```diff
@@ -291,7 +291,7 @@ describe('Entities reducer tests', () => {
       ];
       await store.dispatch(getEntities()).then(() => expect(store.getActions()).toEqual(expectedActions));
     });
-    <%_ if (searchEngine === 'elasticsearch') { _%>
+    <%_ if (searchEngine === 'elasticsearch' || searchEngine === 'couchbase') { _%>
```

Comment:
I think all the conditions could be replaced in a more concise way:
```suggestion
    <%_ if (searchEngine !== false) { _%>
```

Reply:
I agree with you, this how I wanted to implement this. But since you have `searchEngine === 'elasticsearch'''` everywhere, I thought that it was in purpose. If you prefer I could modify all the conditions.

Reply:
`elasticsearch` was the only option available until now, if think that's the condition was like that. However in all locations where the condition could be logically replaced by "is text search available?", I think it would indeed better to modify the condition :+1: